lib/dai-plugin-mcd/src/CdpType.js

```diff
@@ -39,45 +44,46 @@ export default class CdpType {
   }
 
   async getTotalDebt() {
-    const { Art, rate } = await this.ilkInfo();
+    const { Art, rate } = await this._getVat();
     return MDAI.wei(Art)
       .times(rate)
       .shiftedBy(-27);
   }
 
   async getDebtCeiling() {
-    const { line } = await this.ilkInfo();
+    const { line } = await this._getVat();
     return MDAI.rad(line);
   }
 
   async getLiquidationRatio() {
-    const { mat } = await this.ilkInfo('spot');
-    return new BigNumber(mat.toString()).dividedBy(RAY).toNumber();
+    const { mat } = await this._getSpot();
+    const ratio = createCurrencyRatio(USD, MDAI);
+    return ratio(new BigNumber(mat.toString()).dividedBy(RAY).toString());
   }
 
   async getPrice() {
-    const [ilkInfo, liqRatio, par] = await Promise.all([
```

Comment:
it would be better to retain the original name for these variables, since when we use `vat` elsewhere in the code, it refers to the vat contract